pkg/tsdb/influxdb/influxql/response_parser_test.go

```diff
@@ -930,3 +1020,578 @@ func TestResponseParser_Parse(t *testing.T) {
 		})
 	}
 }
+
+func toPtr[T any](v T) *T {
+	return &v
+}
+
+const tableResultFormatInfluxResponse1 = `{
```

Comment:
It would be better to use the pattern we use in prom here for testing with the json test_data files. I don't think it is a blocker on the PR, but we should probably make an issue to update the tests in this DS.

Reply:
That's a good call. I will also refactor the `time_series` parsing since we have a lot of duplicate code with this PR. When I do that, I will refactor testing too. I'll create an issue for myself to track this.

Reply:
https://github.com/grafana/grafana/issues/78550